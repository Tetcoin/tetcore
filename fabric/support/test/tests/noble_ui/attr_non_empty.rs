#[fabric_support::noble [foo]]
mod foo {
}

fn main() {
}
