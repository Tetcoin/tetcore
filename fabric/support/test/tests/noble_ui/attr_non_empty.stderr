error: Invalid noble macro call: expected no attributes, e.g. macro call must be just `#[fabric_support::noble]` or `#[noble]`
 --> $DIR/attr_non_empty.rs:1:26
  |
1 | #[fabric_support::noble [foo]]
  |                          ^^^
